dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -78,7 +78,10 @@
 @NamedQueries({
     @NamedQuery(name = "Entry.getGenericEntryById", query = "SELECT e from Entry e WHERE :id = e.id"),
         @NamedQuery(name = "Entry.getGenericEntryByAlias", query = "SELECT e from Entry e JOIN e.aliases a WHERE KEY(a) IN :alias"),
-        @NamedQuery(name = "io.dockstore.webservice.core.Entry.findCollectionsByEntryId", query = "select new io.dockstore.webservice.core.CollectionOrganization(col.id, col.name, col.displayName, organization.id, organization.name, organization.displayName) from Collection col join col.entries as entry join col.organization as organization where entry.id = :entryId")
+        @NamedQuery(name = "io.dockstore.webservice.core.Entry.findCollectionsByEntryId", query = "select new io.dockstore.webservice.core.CollectionOrganization(col.id, col.name, col.displayName, organization.id, organization.name, organization.displayName) from Collection col join col.entries as entry join col.organization as organization where entry.id = :entryId"),
+        @NamedQuery(name = "getCollectionWorkflows", query = "SELECT new io.dockstore.webservice.core.CollectionEntry(w.id, w.dbUpdateDate, 'workflow', w.sourceControl, w.organization, w.repository, w.workflowName) from BioWorkflow w, Collection col join col.entries as e where col.id = :collectionId and w.id = e.id and w.isPublished = true"),
```

Comment:
There's only 3 other queries in the file and they don't seem to have the same naming convention.

Reply:
My understanding is that JPA named queries should have a namespace of some kind to avoid accidental collisions. 
I don't have a preference between package + class name or just class name since we don't have a `Entry` class in multiple packages

> Do note that query names are part of a global namespace and are not automatically put in any namespace  based on the file they are defined in. In the example above queries are pre-fixed with "Website.", which happens  to be the name of the entity but you can choose anything you want here.

https://arjan-tijms.omnifaces.org/2010/09/where-to-put-named-queries-in-jpa.html